libmozevent/pulse.py

```diff
@@ -137,9 +139,33 @@ def register(self, bus):
                 pulse = None
                 await asyncio.sleep(5)
 
+    def find_matching_queues(self, payload_exchange: str, payload_routes: List[bytes]):
```

Comment:
It'd probably be better to use another of the solutions (especially the multiple channels one), but we can do it in a follow-up. Can you file it?

Reply:
https://github.com/mozilla/libmozevent/issues/33#issuecomment-561139419